examples/src/Email.hs

```diff
@@ -4,6 +4,7 @@
 -- Thanks to @purefn for the help on this!
 
 import Control.Lens
+--import Data.Functor ((<$))
```

Comment:
Please delete this line instead of commenting it out.

Reply:
Good catch. I removed it.